src/components/Book.js

```diff
@@ -1,14 +1,37 @@
 import PropTypes from 'prop-types';
+import '../styles/Book.css';
 
 function Book({ book, handleRemoveBook }) {
-  const { id, title, category } = book;
+  const { title, category } = book;
+
   return (
-    <tr>
-      <td>{id}</td>
-      <td>{title}</td>
-      <td>{category}</td>
-      <td><button type="button" onClick={handleRemoveBook}>Remove book</button></td>
-    </tr>
+    <>
+      <div className="item" id="book">
+        <div className="left">
+          <p className="category">{category}</p>
+          <h3 className="title">{title}</h3>
+          <p className="author">Suzanne Collins</p>
+          <div className="action-buttons">
+            <a className="item-link" href="#book">Comments</a>
+            <a className="item-link remove" onClick={handleRemoveBook} href="#book">Remove</a>
+            <a className="item-link" href="#book">Edit</a>
+          </div>
+        </div>
+        <div className="middle">
+          64%
+          <br />
```

Comment:
- [ ] The progress circle is not present in your design. Kindly add this
*HINT: There are several ways to implement this, you can use tools like [material-ui](https://material-ui.com/components/progress/) to generate this component and pass a random number to indicate different progress for each book.*